src/asmcnc/apps/upgrade_app/screen_upgrade.py

```diff
@@ -196,46 +195,63 @@ def get_correct_unlock_code(self, serial):
     def code_entered(self):
         self.hide_error_message()
         self.show_verifying()
-        self.m.s.write_command('M3 S0')
+        self.m.s.write_command("M3 S0")
         Clock.schedule_once(self.get_restore_info, 0.3)
 
     def get_restore_info(self, dt):
-        self.m.s.write_protocol(self.m.p.GetDigitalSpindleInfo(), "GET DIGITAL SPINDLE INFO")
+        self.m.s.write_protocol(
+            self.m.p.GetDigitalSpindleInfo(), "GET DIGITAL SPINDLE INFO"
+        )
         self.check_info_count = 0
         Clock.schedule_once(self.check_restore_info, 0.3)
 
     def check_restore_info(self, dt):
         self.check_info_count += 1
-        # Value of -999 represents disconnected spindle - if detected then stop waiting
-        if (self.m.s.digital_spindle_ld_qdA != -999 and self.m.s.spindle_serial_number not in [None, -999, 999]) or (self.check_info_count > 10):
+        if (
+            self.m.s.digital_spindle_ld_qdA != -999
+            and self.m.s.spindle_serial_number not in [None, -999, 999]
+            or self.check_info_count > 10
+        ):
```

Comment:
brackets have been changed by the formatter here - do a quick test to ensure that it still behaves as expected/update brackets

Reply:
Very good spot, I've done some testing and it seems like these are logically equivalent because it's evaluated from left to right. But if it were (A and (B or C)) turning into (A and B or C) then it would not be equivalent